### ISSUE TITLE
fastLED.h file missing

### ISSUE THREAD

**Issue by FZRatt** [andrewrbrown/arcade_cabinet_control#1]
Hey Andrew.  When I try to upload your code I get a fatal error. 
 #include <FastLED.h>

          ^~~~~~~~~~~

compilation terminated.

exit status 1
FastLED.h: No such file or directory

What am I missing?  Thanks.

**Comment by andrewrbrown** [comment]
Looks like you are missing the FastLED library.  You can install it by clicking Sketch->Include Library->Manage Libraries
This brings up the library manager.  You can then enter FastLED in the search at the top, then install the library.  

Let me know if that helps.
Thanks,
Andy

**Comment by FZRatt** [comment]
Yep that was it.  This kind of stuff sure makes me feel stupid. lol.  I'm
an IT guy and I can do that stuff in my sleep but electronics just gives me
that deer in the headlights feeling.  Thanks for getting back to me Andrew.

**Comment by andrewrbrown** [comment]
No worries.  I know just enough to be dangerous.  The FastLED library is pretty good.  Make sure you run some of the demo programs from it for fun.

**Comment by FZRatt** [comment]
Do you mind if I ask what colors you chose for your back light?  I'm trying
to use green but it doesn't come up green at all.

**Comment by FZRatt** [comment]
Sorry to keep buggin you man but did you transfer your setup to a circuit
board or did you just leave it all on the breadboard and did you use
resistors in your setup?  I think that is why I can't get green to come up
on my LEDs.  Thanks.  It's too bad Bob doesn't monitor his forums better.
Seems a lot of people struggle with this part.  He could help NOOBS like
myself if he would publish a final wiring diagram.

**Comment by andrewrbrown** [comment]
The motion sensor triggers a pulsing gold glow.  Once the raspberry pi is turned on, it has a white backlight.
What LED's are you using?
    On Friday, May 8, 2020, 10:57:37 PM EDT, FZRatt <notifications@github.com> wrote:  
 
 

Do you mind if I ask what colors you chose for your back light? I'm trying
to use green but it doesn't come up green at all.

**Comment by andrewrbrown** [comment]
I missed you question regarding the breadboard in my last response.  No problem asking more questions.  I'm happy to help.  Bob did a great job getting this project up and running, then moved on.  With the amount of content he puts out, I think he'd have a hard time keeping up with support.  
The breadboard has almost no purpose other than a couple of components.  I bought a tiny breadboard from amazon and 3d printed a bracket to hold everything in one place.  (Arduino, Breadboard, and Relay Mount by kiteflier)  It's not the most robust, but seems to work.  Instead of the breadboard, you could buy a protoboard and solder to it.  No sense making a custom PCB for something this simple.  If you want a bracket for the components, but don't have access to a 3d printer, let me know and I can print and send you one.  It fits a standard Arduino Uno, a micro breadboard, and a two channel relay.

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Arduino, Breadboard, and Relay Mount by kiteflier

I was building an arcade cabinet (https://iliketomakestuff.com/how-to-make-an-arcade-cabinet-part-3/) and wanted...
 |

 |

 |




    On Friday, May 8, 2020, 11:07:30 PM EDT, FZRatt <notifications@github.com> wrote:  
 
 

Sorry to keep buggin you man but did you transfer your setup to a circuit
board or did you just leave it all on the breadboard and did you use
resistors in your setup? I think that is why I can't get green to come up
on my LEDs. Thanks. It's too bad Bob doesn't monitor his forums better.
Seems a lot of people struggle with this part. He could help NOOBS like
myself if he would publish a final wiring diagram.

**Comment by FZRatt** [comment]
Thanks for the offer on the bracket.  I might have to take you up on it.  I
was finally able to get everything working from a prototype perspective
this weekend to my amazement.  I just need to get it transferred to the
arcade now.  I still can't figure out why Bob is using a resistor on the
mode switch.

**Comment by andrewrbrown** [comment]
Let me know on the bracket.  We'd have to figure out a way to take this discussion offline from GitHub to get your mailing address if you want me to send you one.
Bob had a couple big mistakes in his schematic.  You're using my schematic, correct?  
My son has decided Centipede is his favorite now, and Galaga has lost favor.  Now I regret not adding a track ball control...  This is such a great project.
Thanks,Andy
    On Monday, May 11, 2020, 10:35:08 AM EDT, FZRatt <notifications@github.com> wrote:  
 
 

Thanks for the offer on the bracket. I might have to take you up on it. I
was finally able to get everything working from a prototype perspective
this weekend to my amazement. I just need to get it transferred to the
arcade now. I still can't figure out why Bob is using a resistor on the
mode switch.

**Comment by FZRatt** [comment]
My email address is fj40ratt@gmail.com.  Happy to move it out of github.
Galaga has been my favorite game forever.  Centipede will never take over
lol.  I actually got Bob's schematic to work believe it or not.
